activesupport/test/constantize_test_cases.rb

```diff
@@ -73,6 +73,10 @@ def run_constantize_tests_on
         yield("RaisesNoMethodError")
       end
     end
+
+    with_autoloading_fixtures do
+      yield('Prepend::SubClassConflict')
+    end
```

Comment:
👍  Code climate is complaining because it wants single quotes to be doubles on both lines.